src/style.css

```
@@ -0,0 +1,67 @@
+* {
+  margin: 0;
+}
```

don't reset for all. It's very difficult  process 
```suggestion
* {
  box-sizing: border-box;
}

ul, li, a {
  margin: 0;
}
```